reactor-pool/src/test/java/reactor/pool/CommonPoolTest.java

```diff
@@ -1117,6 +1119,31 @@ void discardCloseableWhenCloseFailureLogs(PoolStyle configAdjuster) {
 		}
 	}
 
+	@ParameterizedTestWithName
+	@MethodSource("allPools")
+	void pendingTimeout(PoolStyle configAdjuster) throws Exception {
+		PoolBuilder<String, ?> builder = PoolBuilder
+				.from(Mono.just("pendingTimeout"))
```

Comment:
The "pendingTimeout" is confusing here -> consider calling it "pooled" for instance.

Reply:
done